main/src/cgeo/geocaching/utils/CalculationUtils.java

```diff
@@ -96,13 +96,13 @@ boolean eat(final int charToEat) {
                     }
                     final String func = str.substring(startPos, this.pos);
                     x = parseFactor();
-                    if (func.equals("sqrt")) {
```

Comment:
Do we use Java 6 or 7? In case of 7 this could be replaced by a single switch statement.